.github/workflows/main.yml

```diff
@@ -33,6 +33,9 @@ jobs:
       - name: Install rust toolchain
         run: rustup show
 
+      - name: Install libgcc1
```

Comment:
There was one stage during the development,  I was trying to build a `RUST_TARGET` instead of `MUSL_TARGET` of the connector-proxy to support unit tests triggered by `make go-test-ci`. And installed this library to support the built rust target.

However, later I decided to move `make go-test-ci` to stage 2, in which both flowctl as a RUST target and `connector-proxy` as a MUSL target are available. And this is no longer needed.